### DISCUSSION TITLE
In App router, can use same tag for multiple fetch request to revalidatetag for multiple request?

### DISCUSSION THREAD

**Discussion by bharath-hebbar** [vercel/next.js#70188] [Help] [answered]
### Summary

Can we use same tag in fetch for multiple request to revalidate that tag?

ex: `getA(){ fetch('test.com/a', {next: {tag: [ 'test']} }) };
 getB(){ fetch('test.com/b', {next: {tag: [ 'test']} }) }
`

after to revalidate both:

`revalidateTag('test')`

**Comment by Juneezee** [comment, marked as answer]
It is not recommended to reuse only one tag for multiple uses. You can try:

```javascript
fetch("test.com/a", { next: { tags: ["test", "a"] } });

fetch("test.com/b", { next: { tags: ["test", "b"] } });
```

and then 

```javascript
revalidateTag('test')
```

will purge cache for both `test.com/a` and `test.com/b`.

See https://nextjs.org/docs/app/building-your-application/caching#fetch-optionsnexttags-and-revalidatetag for more information
